source/arm9/image.c

```diff
@@ -83,21 +87,23 @@ void image8to16trans(sImage *img, u8 transparentColor)
 
     img->bpp = 16;
     img->image.data16 = temp;
+
+    return true;
 }
 
-void imageTileData(sImage *img)
+bool imageTileData(sImage *img)
 {
     // Can only tile 8 bit data that is a multiple of 8 in dimention
-    if (img->bpp != 8 || (img->height & 3) != 0 || (img->width & 3) != 0)
```

Comment:
Also, this check seems to have been entirely wrong?

Reply:
It looks like it.